boards/nordic/nrf54l15pdk/nrf54l15_cpuapp_common.dtsi

```diff
@@ -6,7 +6,7 @@
 
 /* This file is common to the secure and non-secure domain */
 
-#include <nordic/nrf54l15_cpuapp.dtsi>
+#include <nordic/nrf54l15_cpuapp_enga.dtsi>
```

Comment:
```suggestion
#include <nordic/nrf54l15_enga_cpuapp.dtsi>
```
To be compatible with Kconfigs `CONFIG_SOC_NRF54L15_ENGA_CPUAPP`?

Reply:
done